.github/workflows/standard_spa_pr_NPM.yml

```diff
@@ -353,7 +353,7 @@ jobs:
     container:
       image: cypress/browsers:node18.12.0-chrome106-ff106
       options: --platform linux/amd64 --privileged
-    timeout-minutes: 15
+    timeout-minutes: 25
```

Comment:
Addresses the following error `The job running on runner jupiterone-dev-runner-amd64-spms6-0 has exceeded the maximum execution time of 15 minutes`